torch/csrc/utils/pybind.cpp

```diff
@@ -7,8 +7,14 @@ namespace detail {
 
 bool type_caster<c10::SymInt>::load(py::handle src, bool) {
   if (torch::is_symint(src)) {
+    auto node = src.attr("node");
```

Comment:
Can you explain under what circumstance the `node` is a py-bound SymNode vs non-py-bound SymNode?